specs/endpoints/meal_index.spec.js

```diff
@@ -0,0 +1,30 @@
+var shell = require('shelljs');
+var request = require("supertest");
+var app = require('../../app');
+
+describe('Meal index API', () => {
+  beforeAll(() => {
+    shell.exec('npx sequelize db:create')
+  });
+  beforeEach(() => {
+    shell.exec('npx sequelize db:migrate:undo:all')
+    shell.exec('npx sequelize db:seed:undo:all')
+    shell.exec('npx sequelize db:migrate')
+    shell.exec('npx sequelize db:seed:all')
+  });
+  afterEach(() => {});
+
+  describe('Test GET /api/v1/meals', () => {
```

Comment:
This describe should be moved above the setup/teardown and that should be replaced with the update version. I will make this update along with my other test updates.

Reply:
Sounds good.